experiments/ssl_experiments/pimodel_mcdropout_cifar10.py

```diff
@@ -61,16 +71,18 @@ def add_model_specific_args(parent_parser):
 
     active_set = ActiveLearningDataset(
         CIFAR10(params.data_root, train=True, transform=PIModel.train_transform, download=True),
-        pool_specifics={'transform': PIModel.test_transform},
-        make_unlabelled=lambda x: x[0])
-    active_set.label_randomly(100)
+        pool_specifics={'transform': PIModel.test_transform})
+    active_set.label_randomly(500)
 
     print("Active set length: {}".format(len(active_set)))
     print("Pool set length: {}".format(len(active_set.pool)))
 
-    heuristic = get_heuristic(params.heuristic, params.shuffle_prop)
-    net = vgg16(pretrained=False, num_classes=10)
-    model = PIActiveLearningModel(network=net, active_dataset=active_set, hparams=params)
+    heuristic = get_heuristic(params.heuristic)
+    model = vgg16(pretrained=False, num_classes=10)
+    weights = load_state_dict_from_url('https://download.pytorch.org/models/vgg16-397923af.pth')
+    weights = {k: v for k, v in weights.items() if 'classifier.6' not in k}
```

Comment:
What does this do ? Should there be a comment ?

Reply:
Finetuning from imagenet is much more stable for AL on CIFAR10.

Reply:
Ok, forgot using pretrained=True was only good for 1000 classes.